shopinvader_delivery_pickup/services/delivery_pickup.py

```diff
@@ -10,7 +10,7 @@
 class DeliveryPickupService(Component):
     _inherit = "base.shopinvader.service"
     _name = "shopinvader.delivery.pickup.service"
-    _usage = "delivery_pickup"
+    _usage = "delivery_pickups"
```

Comment:
@sebastienbeau Why do you rename my entry points!!! You'll break my existing websites. IMO, endpoint name must be singular not plural